src/common/queries/useContractFtMetadata.ts

```diff
@@ -1,12 +1,12 @@
 import { useQuery } from '@tanstack/react-query';
 
-import { useApi } from '../api/useApi';
+import { useMetadataApi } from '../api/useApi';
 
 export function useContractFtMetadata(contractId?: string) {
-  const api = useApi();
+  const tokenMetadataApi = useMetadataApi();
   return useQuery({
     queryKey: ['contract-ft-metadata', contractId],
-    queryFn: () => api.tokenMetadataApi?.getFtMetadata(contractId!),
+    queryFn: () => tokenMetadataApi?.getFtMetadata(contractId!),
```

Comment:
Is the metadata api going to transistion to the endpoint model too?

Reply:
not for now, current version still follows the old convention